### PR TITLE
made changes in packing slip files

### PR DESCRIPTION
**Pull request by shantanu-kadam-epi**
> Please provide enough information so that others can review your pull request:

> Explain the **details** for making this change. What existing problem does the pull request solve?

> Screenshots/GIFs

### CODE REVIEW

**Comment by shantanu-kadam-epi** [comment]
Asana Ticket Link: https://app.asana.com/0/1200151327710888/1200022610032012

File: erpnext/stock/doctype/packing_slip/packing_slip.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ cur_frm.cscript.validate_case_nos = function(doc) {
 		doc.to_case_no = doc.from_case_no;
 		refresh_field('to_case_no');
 	} else if(cint(doc.to_case_no) < cint(doc.from_case_no)) {
-		frappe.msgprint(__("'To Case No.' cannot be less than 'From Case No.'"));
+		frappe.msgprint(__("'To Package No.' cannot be less than 'From Package No.'"));
```
Comment (hrwX) [inline erpnext/stock/doctype/packing_slip/packing_slip.js:95]:
```suggestion
		frappe.msgprint(__("<b>To Package No.</b> cannot be less than <b>From Package No.</b>"));
```

File: erpnext/stock/doctype/packing_slip/packing_slip.py [hunk 2]
```diff
@@ -47,11 +47,11 @@ def validate_case_nos(self):
 			Validate if case nos overlap. If they do, recommend next case no.
 		"""
 		if not cint(self.from_case_no):
-			frappe.msgprint(_("Please specify a valid 'From Case No.'"), raise_exception=1)
+			frappe.msgprint(_("Please specify a valid 'From Package No.'"), raise_exception=1)
 		elif not self.to_case_no:
 			self.to_case_no = self.from_case_no
 		elif cint(self.from_case_no) > cint(self.to_case_no):
-			frappe.msgprint(_("'To Case No.' cannot be less than 'From Case No.'"),
+			frappe.msgprint(_("'To Package No.' cannot be less than 'From Package No.'"),
```
Comment (hrwX) [inline erpnext/stock/doctype/packing_slip/packing_slip.py:54]:
```suggestion
			frappe.msgprint(_("<b>To Package No.</b> cannot be less than <b>From Package No.</b>"),
```

**Review by hrwX — CHANGES_REQUESTED** [review]

**Comment by shantanu-kadam-epi** [comment]
Closing this, New pull request created.
https://github.com/Bloomstack/erpnext/pull/1185

**Pull request closed without merging**